<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFile">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outDir">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>54</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>54</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>输出目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>解压</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearcache">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除缓存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readfile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>读取文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>解析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openout">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输出文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearout">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空输出</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="firstasname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>将第一行作为名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="disableupload">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>禁用图片上传</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="h1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>#</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="h2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>##</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="h3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>###</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="h4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>####</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="h5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>#####</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H1">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H2">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>H2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H3">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>H3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H4">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>240</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>H4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H5">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>H5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="unsettitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>295</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>没有标题时将第一行作为名称</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useopf">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>325</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>使用opf生成文件序列</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="connectedopf">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接式opf</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyID">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>265</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="keyid">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>265</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>KeyId</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>295</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="secret">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>295</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Secret</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>325</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="endpoint">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>326</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>EndPoint</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>355</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bucket">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>355</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bucket</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>385</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="path">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>385</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>阿里云OSS配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
